src/mudlet.pro

```diff
@@ -111,19 +111,23 @@ VERSION = 4.17.2
 
 # if you are distributing modified code, it would be useful if you
 # put something distinguishing into the MUDLET_VERSION_BUILD environment
-# variable to make identification of the used version simple
-# the qmake BUILD variable is NOT built-in
-BUILD_TEST = $$(MUDLET_VERSION_BUILD)
-isEmpty( BUILD_TEST ) {
+# variable (it should use '-' as the first character) to make identification of
+# the used version simple the qmake BUILD variable is NOT built-in
+BUILD = $$(MUDLET_VERSION_BUILD)
+isEmpty( BUILD ) {
 # Possible values are:
 # "-dev" for the development build
 # "-ptb" for the public test build
 # "" for the release build
-   BUILD_TEST = "-dev-"$${GIT_SHA1}
+   BUILD = "-dev-"$${GIT_SHA1}
 } else {
-   BUILD_TEST = $${GIT_SHA1}
+   BUILD = $${BUILD}-$${GIT_SHA1}
```

Comment:
I previously had not realised I could append something to a qmake variable like this - but it is needed so that a supplied `MUDLET_VERSION_BUILD` environmental variable gets into the version string.